recipes/Libriheavy/ASR/transformer/README.md

```diff
@@ -0,0 +1,52 @@
+# Libriheavy ASR with Transformers.
+This folder contains the scripts to train a Transformer-based speech recognizer.
+
+1. Please download Libri-Light at https://github.com/facebookresearch/libri-light/tree/main/data_preparation
+After this step, please make sure you have all the splits (small, medium, and large) in one folder.
+
+2. Please prepare Libriheavy manifest following the instruction at https://github.com/k2-fsa/libriheavy
+After this step, please make sure you have all the "jsonl.gz" Libriheavy manifest files in one folder.
+
+# How to run
+```shell
+python train.py hparams/transformer.yaml --data_folder=/path/to/Libri-Light --manifest_folder=/path/to/Libriheavy
+
+```
+
+# LibriSpeech Dev/Test Results
+Results of Trained with Libriheavy and test with LibriSpeech.
+
+| Release | hyperparams file | Dev Clean WER (Transformer LM) | Test Clean WER (Transformer LM) | Test Other WER (Transformer LM) | HuggingFace link | Model link | GPUs |
+|:-------------:|:-------------:|:-------------:|:---------------------------:| :-----:| :-----:| :-----:| :--------:|
+| 24-12-09 | conformer_large.yaml | 1.58 | 1.74 | 3.92 | Not Avail. | Not Avail. | 8xA100 80GB |
```

Comment:
I think for the results we need to clarify which training set are you using. Furthermore, I know as a Samsung employee you cannot release models (what a shame, you need to change things internally! Open weights is the bare minimum for OS AI applications...), but I might have to train a model. For the last couple of recipes that we merged, we didn't released any models... I guess that for this one it would be nice to have something to display, maybe a streaming model? CC: @TParcollet

Reply:
@Adel-Moumen Hi Adel, thank you very much for your comments! I really appreciate it. I have talked with the boss in our centre and was told Samsung may view models trained within the centre as Samsung IP, and I need wait for further clarifications on this. Once I've heard a definite answer from the boss, I'll let you now. I'll address the other comments. Thank you.

Reply:
Hi @shucongzhang, yeah I know that for a company it can be tricky to release models. Anyway, I might train a model using your recipe. I just need to think in terms of compute resources.